bundles/AdminBundle/Resources/public/js/pimcore/element/selector/asset.js

```diff
@@ -225,6 +233,18 @@ pimcore.element.selector.asset = Class.create(pimcore.element.selector.abstract,
                 }
             ];
 
+            if (this.parent.multiselect) {
+                columns.unshift(
+                    {
+                        xtype: 'checkcolumn',
+                        fieldLabel: '',
+                        name: 'asset-select-checkbox',
+                        text: t("select"),
+                        dataIndex : 'asset-selected'
```

Comment:
```suggestion
                        dataIndex : 'asset-selected',
                        sortable: false
```

Set sortable to false to prevent an error message for now